src/classlibnative/inc/nlsinfo.h

```diff
@@ -145,53 +86,7 @@ class COMNlsInfo {
 
     static void QCALLTYPE nativeNormalizationInitNormalization(int NormForm, BYTE* pTableData);
```

Comment:
@jkotas we have some special case for normalization when we run on pre-Windows 8 (i.e. Win7). we carry the normalization data and use it when running on Windows 7. the reason is the normalization data on Windows 7 was not that good. if we are willing to scarify that when running on Windows 7, I can get rid of the fcalls along with the data too which will save us some more size.